src/main/java/core/basesyntax/Model/Cat.java

```diff
@@ -1,4 +1,4 @@
-package core.basesyntax.Model;
+package core.basesyntax.model;
```

Comment:
you also should rename the package
<img width="216" alt="image" src="https://user-images.githubusercontent.com/96740815/215719538-cd79a3b6-6e04-4c91-89b8-f886fcdaf5f7.png">